www/_assets/css/screen.css

```diff
@@ -716,6 +716,16 @@ section#archive h3.chpts a.social:after {
 section#archive .chapter_location h3.chpts {
   overflow: hidden;
 }
+.logo-container img {
+  max-height: 100px;
+  max-width: 100px;
```

Comment:
Sorry to be a nitpick - this still misaligns those that are using the wordmark logo (e.g. `pyladies_sd_brno.png`). For example:

![2015-11-13 at 5 23 pm](https://cloud.githubusercontent.com/assets/1191069/11151060/5383e34e-8a2b-11e5-9305-f786531f6a6d.png)

To fix this, I think instead of `max-height` and `max-width` for `.logo-container img` and for `section#archive .chapter_location img`, it should actually be just `height` and `width` set to `100px`. Sure, the logo would look stretched but I think that is okay compared to misalignment.

I hope that makes sense.